pkg/storaged/lvol-tabs.jsx

```diff
@@ -107,15 +107,15 @@ function lvol_and_fsys_resize(client, lvol, size, offline, passphrase) {
             if (size - crypto_overhead > vdo.physical_size)
                 return vdo.grow_physical();
             else if (size - crypto_overhead < vdo.physical_size)
-                return cockpit.reject(_("VDO backing devices can not be made smaller"));
+                return Promise.reject(_("VDO backing devices can not be made smaller"));
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18487-20230310-143650-f841503b-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)